money-api/src/test/scala/com/comcast/money/api/DefaultSpan.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2012-2015 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.api
+
+import java.lang
+import java.lang.{ Double, Long }
+
+import scala.collection._
+import scala.collection.JavaConversions._
+
+// obviously not threadsafe and very mutable, basing this kinda on what I did for the java-only core
+class DefaultSpan(
+    val id: SpanId,
+    val spanName: String,
+    spanMonitor: SpanMonitor,
+    notes: mutable.Map[String, Note[_]] = new mutable.HashMap[String, Note[_]],
+    propagate: Boolean = false
+) extends Span {
+
+  // TODO: yea, hacky, just prototyping though
+  private var stopTime: Long = _
+  private var stopInstant: Long = _
+  private var startedTime: Long = _
+  private var startInstant: Long = _
+  private var success: Boolean = true
+
+  override def startTime(): Long = startedTime
+
+  override def start(): Unit = {
+    startedTime = System.currentTimeMillis
+    startInstant = System.nanoTime
+
+    // Start watching me
+    spanMonitor.watch(this)
```

Comment:
wrt the existing implementation, I think we will have a `money-akka` or similarly named module that will be the current implementation.  However, I think there is value in having a `money-scala` that doesn't rely on Akka as the implementation.  Akka has historically had version / upgrade issues.

Once we are ok with the API, the next step is to "port" the current implementation as well as get started on a `money-scala` module that can avoid akka and thread local things.  That will allow us to continue to iterate on the 0.9 API.